Dockerfile.migrate

```
@@ -0,0 +1,16 @@
+FROM docker.io/library/rust:1.74.0-buster as builder
+
+RUN apt update && apt install build-essential cmake -y
+RUN cargo install sqlx-cli --no-default-features --features native-tls,postgres --root /app/target/release/
```

We definitely want to ping the version here, as a bug in a new release could nuke a production database. Hopefully that syntax works?
```suggestion
RUN cargo install sqlx-cli@v0.7.3  --no-default-features --features native-tls,postgres --root /app/target/release/
```

---

Yeah, good point. 

Tested locally and it seems like the syntax is without 'v':

```
cargo install sqlx-cli@v0.7.2                                                                                  
error: invalid value 'sqlx-cli@v0.7.2' for '[CRATE[@<VER>]]...': unexpected character 'v' while parsing major version number
```